azure/durable_functions/models/TaskOrchestrationExecutor.py

```diff
@@ -167,7 +167,7 @@ def parse_history_event(directive_result):
             # will be passed directly to the orchestrator as the output of some activity
             if event_type == HistoryEventType.SUB_ORCHESTRATION_INSTANCE_COMPLETED:
                 return json.loads(directive_result.Result, object_hook=_deserialize_custom_object)
-            if event_type == HistoryEventType.TASK_COMPLETED:
+            if event_type == HistoryEventType.TASK_COMPLETED and directive_result.Result is not None:
                 return json.loads(directive_result.Result, object_hook=_deserialize_custom_object)
             if event_type == HistoryEventType.EVENT_RAISED:
```

Comment:
Should we make the same `None` checks for the other two `if` conditions? It seems possible that we'd run into the same problem if either a sub-orchestration returned `None` or if an external event was raised with a `None` value for the input.